Makefile.linux

```diff
@@ -56,8 +60,6 @@ LINUX_FLAGS += 	-fno-asynchronous-unwind-tables \
 		-fno-stack-protector \
 		-ffunction-sections \
 		-fdata-sections \
-		-fno-defer-pop \
-		-Wno-pointer-sign \
```

Comment:
they are not supported in the clang on mac, i don't know if there are equivalents we can replace with, if we do wanna keep these, i'll put a ifdef there